x/incentives/keeper/distribute_test.go

```diff
@@ -1409,6 +1410,20 @@ func (s *KeeperTestSuite) TestSyncVolumeSplitGroup() {
 
 			expectedError: types.CumulativeVolumeDecreasedError{PoolId: uint64(2), PreviousVolume: osmomath.NewInt(200), NewVolume: osmomath.NewInt(100)},
 		},
+
+		"total weight is zero due to no records": {
+			groupToSync: types.Group{
+				GroupGaugeId:      defaultGroupGaugeId,
+				InternalGaugeInfo: types.InternalGaugeInfo{},
+				SplittingPolicy:   types.ByVolume,
+			},
+			updatedPoolVolumes: []osmomath.Int{
+				osmomath.ZeroInt(),
+				osmomath.ZeroInt(),
+			},
+
+			expectedError: types.GroupTotalWeightZeroError{GroupID: defaultGroupGaugeId},
+		},
```

Comment:
Side question, if you had two pool gauges for two CL pools with zero liquidity, would the total weight also be zero?

Reply:
Yes. We would like to prevent the creation of such groups in the first place by attempting to sync weights at the time of group creation:
https://github.com/osmosis-labs/osmosis/issues/6581